client/src/components/UserAuth.vue

```diff
@@ -0,0 +1,166 @@
+<template>
+  <form @submit.prevent="submitForm" class="userLogin__form">
+    <label for="username">E-mail:</label>
+    <input
+      type="email"
+      name="email"
+      id="email"
+      placeholder="Type your email"
+      class="userLogin__input"
+      v-model.trim="email"
+    />
+
+    <label for="password">Password:</label>
+    <input
+      type="password"
+      name="password"
+      placeholder="Type your password"
+      class="userLogin__input password__input"
+      v-model.trim="password"
+    />
+    <p v-if="!formIsValid" class="userLogin__notification">
+      Please enter a valid email or password (must be at least 6 characters
+      long).
+    </p>
+
+    <button class="submit__btn-active">{{ changeSubmitBtnValue }}</button>
+    <button
+      type="button"
+      class="submit__btn"
+      mode="submit__btn"
+      @click="switchBtnMode"
+    >
+      {{ changeBtnMode }}
+    </button>
+  </form>
+</template>
+
+<script>
+export default {
+  data() {
+    return {
+      email: "",
+      password: "",
+      formIsValid: true,
+      btnMode: "login",
+    };
+  },
+  methods: {
+    submitForm() {
+      console.log("submit");
+      this.formIsValid = true;
+      if (this.email === "" || this.password.length < 6) {
+        this.formIsValid = false;
+        return;
+      } else {
+        console.log(this.email, this.password);
+        //send http
+      }
+    },
+    switchBtnMode() {
+      if (this.btnMode === "login") {
+        this.btnMode = "signUp";
+      } else {
+        this.btnMode = "login";
+      }
+    },
+  },
+  computed: {
+    changeSubmitBtnValue() {
+      if (this.btnMode === "login") {
+        return "Login";
+      } else {
+        return "SignUp";

```

Comment:
- Ločil sem na client & server side folder, kot dva ločena repota znotraj enega repota
- dodal sem nov auth route
- naredil sem basic template in metodo za submitanje ter basic styling

Reply:
super..